.gitignore

```diff
@@ -171,3 +171,5 @@ cython_debug/
 /src/*/libs/**/__pycache__/
 /src/*/libraries/libs_39_32/**/__pycache__/
 /src/*/libraries/libs_39_64/**/__pycache__/
+
+/src/Switch-FTDI_FTD245XX/thermusb_example
```

Comment:
Is this an example that should go into the examples folder? Then it should be moved there and does not need to be ignored.

Otherwise it looks really strange like "Hey, we have an example, but we don't show it."
Also need to consider, what's excluded from the repository is not saved in case your local files get deleted.